src/components/lineChart.js

```diff
@@ -39,6 +45,7 @@ class Chart extends React.Component {
   }
 
   componentDidUpdate() {
+    console.log(this.state.myVegaSpec)
```

Comment:
Do we use console.log in production code? I assume not.

BTW: what is best logging framework for JS?